src/main/resources/reference.conf

```diff
@@ -154,40 +155,44 @@ akka-kryo-serialization {
 
   optional-basic-mappings {
```

Comment:
add comment like:
# mappings are part of the API - consider semantic version when changing